en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-250-to-320.md

```diff
@@ -2611,7 +2611,7 @@ Follow the instructions below to configure WSO2 API Manager for the WSO2 API-M A
         password = "password"
         ```
 
-    -   The second datasource points to the **WSO2 Data Analytics (WSO2 DAS) based previous datasource WSO2AM_STATS_DB for statistics** .
+    -   The second datasource points to the **WSO2 Data Analytics (WSO2 DAS) based newly migrated WSO2AM_STATS_DB for statistics**.
```

Comment:
This chane is not required , as we use the term stat db only for previous versions analytics database and it should be the previous versions datasouce. The issue is only in AM datasource

Reply:
Fixed with 6e72fda